### DISCUSSION TITLE
Release branch

### DISCUSSION THREAD

**Discussion by joshua-white** [GEOS-DEV/GEOS#1645] [Usability]
The code is getting mature enough we should start planning ahead for a more complicated Git model, with separate release and develop branches.  Our issue keeping the tutorials up to date is a good example/motivator.  I don't think we need to do this today, but we should think a bit about how we want to handle release cycles going forward.  Thoughts?

**Comment by rrsettgast** [comment]
We can just go back to the gitflow model...or just a straight OneFlow...or we can actually follow through with the modified OneFlow that I had proposed be our model.

**Comment by paveltomin** [comment]
Seems already implemented with tags https://github.com/GEOS-DEV/GEOS/tags ?
